src/main/scala/ChatBot.scala

```diff
@@ -0,0 +1,27 @@
+class ChatBot {
+
+  var open = true
+  var angryCount = 0
+
+  def playChatBot = {
+
+    while (open == true) {
+      val rand = scala.util.Random
```

Comment:
you're calling this in one place, as well as you can call it directly to get the value you want so shoudn't really need it as a variable